src/term/mod.rs

```diff
@@ -988,6 +990,11 @@ impl Term {
     ) -> RenderableCellsIter {
         let selection = selection.and_then(|s| s.to_span(self))
             .map(|span| span.to_range());
+        let cursor = if window_focused {
```

Comment:
Unrelated to this PR

Reply:
These changes were made to reduce the amount of parameters of the method below clippy's threshold.